github-actions/check-data-assets/README.md

```diff
@@ -19,7 +32,11 @@ Options:
                                   
                                   For protobuf/avro/json_schema the check will
                                   be performed for all file(s)  [required]
-  Database Options:               Options for checking contract complaince for
+  -o, --output [text|json|markdown]
+                                  Format of the output. Options are: text
+                                  (default), json, or markdown which is
+                                  intended to be used as a PR comment
```

Comment:
super nit: needs a period after "PR comment"

Reply:
Just copying and pasting from [here](https://github.com/gabledata/product/blob/main/cli/gable/commands/data_asset.py#L263) so I don't want to make the change here, and I don't want to make a PR to add just a period :D

But if you see it again over in product feel free to sneak that into another PR 👍